File: src/main/kotlin/movieRiew/utilities/Tools.java
package movieRiew.utilities;

import java.text.SimpleDateFormat;
import java.util.Date;
import java.util.Random;
import java.util.UUID;

import static java.lang.Math.floor;

public class Tools {

    public static boolean hasDecimal(float number) {
        float roundedNumber = (float) floor(number);
        return number - roundedNumber > 0;
    }

    public static String dateFormat(long timestamp) {
        SimpleDateFormat dateFormat = new SimpleDateFormat("d/M/yyyy");
        return dateFormat.format(new Date(timestamp * 1000));
    }

    public static long currentDate() {
        return new Date().getTime() / 1000;
    }

    public static long randomDate() {
        Random random = new Random(System.currentTimeMillis());
        long sixtyDaysAgoMs = 60 * 24 * 60 * 60 * 1000L;

        long randomMs = random.nextLong(sixtyDaysAgoMs);
        return (System.currentTimeMillis() - sixtyDaysAgoMs + randomMs) / 1000;
    }

    public static void newPage() {
        System.out.println("\n".repeat(60));

    }

    public static String uuid() {
        return UUID.randomUUID().toString();
    }





}

File: src/main/kotlin/fo/utilities/Tools.kt
package fo.utilities

import java.text.SimpleDateFormat
import java.util.*
import kotlin.math.floor
import kotlin.random.Random

class Tools {
    companion object
    {

        fun hasDecimal(number: Float): Boolean {
            val roundedNumber = floor(number)
            return number - roundedNumber > 0
        }

        fun dateFormat(timestamp: Long): String
        {
            val dateFormat = SimpleDateFormat("d/M/yyyy")
            return dateFormat.format(Date(timestamp * 1000) )

        }

        fun currentDate(): Long
        {
            return Date().time / 1000
        }

        fun randomDate(): Long
        {
            val random = Random(System.currentTimeMillis()) // Create a new Random object
            val sixtyDaysAgoMs = 60 * 24 * 60 * 60 * 1000L // Calculate 60 days ago in milliseconds

            val randomMs = random.nextLong(sixtyDaysAgoMs) // Generate a random number of milliseconds within the time range
            return (System.currentTimeMillis() - sixtyDaysAgoMs + randomMs) / 1000


        }

        fun uuid(): String
        {
            return UUID.randomUUID().toString()
        }

    }


}